.babelrc

```diff
@@ -1,3 +1,14 @@
 {
-  "presets": ["react-native"]
+  "presets": [
+    [
+      "env",
```

Comment:
This is the modern replacement fro es2015 preset. It autodetects the features to be transpiled / polyfilled based on the target platforms.